### PR TITLE
Purge obsolete code

### PR DESCRIPTION
**Pull request by canndrew**
Remove code which has been made obsolete by our switch to the p2p crate.

### CODE REVIEW

File: src/net/listener.rs [hunk 1]
```diff
@@ -259,28 +258,31 @@ mod test {
...
-            .and_then(|(mc, listeners, listener0, addrs0)| {
+            .and_then(move |listener0| {
+                let addr0 = listener0.addr();
+                let addrs0 = {
+                    unwrap!(addr0.expand_local_unspecified())
+                    .into_iter()
+                    .collect::<HashSet<_>>()
+                };
                 let (addrs, addrs_rx) = listeners.addresses();
+                println!("addrs0: {:?}", addrs0);
```
Comment (povilasb) [inline src/net/listener.rs]: I guess prints were left accidentially :)
Remove them?

**Comment by povilasb** [comment]
In the future don't forget our conventions https://github.com/autumnai/leaf/blob/master/CONTRIBUTING.md#commit-message-format to stay consistent :)

**Comment by canndrew** [comment]
> In the future don't forget our conventions
Oh, I had forgotten! Thanks.

I'll squash the commits and rename them.

File: src/net/listener.rs (hunk 1 above)
Comment (canndrew) [reply src/net/listener.rs]: Whoops :p

**Pull request merged**